src/sentry/issues/auto_source_code_config/task.py

```diff
@@ -62,22 +64,30 @@ def process_event(project_id: int, group_id: int, event_id: str) -> None:
         logger.error("Event not found.", extra=extra)
         return
 
-    stacktrace_paths = identify_stacktrace_paths(event.data)
-    if not stacktrace_paths:
+    if not event_can_be_processed(event):
```

Comment:
`event_can_be_processed` will now be the shared function between this code and post_process.